1-mandatory/2-people-i-know.js

```diff
@@ -393,17 +393,38 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+
+let friendsWhoAreColleaguesOfStacie = friends.filter(friend => friend.colleagues.some(colleague => colleague.name === 'Stacie Villarreal'))
+  .map(friend => friend.name.first + ' ' + friend.name.last);
+
```

Comment:
Well done => I like how .some, are used here. Also, I've seen another one using .find, which is the same